src/dmd/mars.d

```diff
@@ -1557,7 +1559,7 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
     for (size_t i = 1; i < arguments.dim; i++)
     {
         const(char)* p = arguments[i];
-        const(char)[] arg = p.toDString();
+        const(char[]) arg = p.toDString();
```

Comment:
Why? Seems unrelated, unless you want to put the constraint that it is not reassign, in which case use `const` ?

Reply:
I did this mainly for myself, to keep me from reassigning a slice of `arg` to `arg` itself and messing up something in another place. Probably no longer useful now that the rework is done.